### PR TITLE
Updated documentation.

### PR DESCRIPTION
**Pull request by zivy**
Improving docstrings and added a readme.

### CODE REVIEW

**Review by dave3d — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# SimpleITKUtilities
+
+![SimpleITK Utilities Testing](https://github.com/SimpleITK/SimpleITKUtilities/actions/workflows/main.yml/badge.svg)&nbsp;&nbsp;[![SimpleITK Utilities Website](https://img.shields.io/website-up-down-brightgreen-red/http/shields.io.svg)](http://simpleitk.org/SimpleITKUtilities/)
+
```
Comment (blowekamp) [inline README.md:4]: Need to link to the Sphinx docs: http://simpleitk.org/SimpleITKUtilities/

Should probably link it SimpleITK/ITK too.

File: SimpleITK/utilities/fft.py [hunk 2]
```diff
@@ -22,20 +22,22 @@
...
 
     If the moving image grid is not congruent with fixed image ( same origin, spacing and direction ), then it will be
     resampled onto the grid defined by the fixed image.
 
     Efficiency can be improved by reducing the resolution of the image or using a projection filter to reduce the
     dimensionality of the inputs.
 
-    :param fixed: A SimpleITK image object
+    :param fixed: A SimpleITK image object.
+    :type fixed: sitk.Image
```
Comment (blowekamp) [inline SimpleITK/utilities/fft.py:35]: Duplication should be avoided. The arguments are already typed in the signature. Perhaps this module would help:
sphinx-autodoc-typehints

**Review by blowekamp — APPROVED** [review]
Good changes over all. Thanks! 

Mainly something to add to the toplevel readme, and need to determine what is the best way to handle argument types.

File: SimpleITK/utilities/fft.py (hunk 2 above)
Comment (zivy) [reply SimpleITK/utilities/fft.py]: types removed from docstring

**Pull request merged**